build_all/docs/Doxyfile

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           = "Microsoft Azure IoT Device SDK for C"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = 1.3.9
+PROJECT_NUMBER         = 1.6.0
```

Comment:
Shouldn't this be something higher? We are already at 1.6.0 in `master`.

Reply:
For public-preview we usually keep the same version as in master. Maybe another point to fix on the versioning strategy?